### PR TITLE
rpcserver: expose graph tlv

### LINKED ISSUE #7080: [feature]: expose tlv data for gossip messages in `DescribeGraph`
TLV extensions for gossip messages are received by `lnd` and stored on disk, but there is no way to retrieve that data through the api. This issue is about extending `DescribeGraph` and other relevant graph calls with a `tlv_records` field.

Example application: https://github.com/mempool/mempool/issues/2658

### PR DESCRIPTION
**Pull request by joostjager**
Exposes graph tlv data that is already stored in the database.

Fixes #7080

### CODE REVIEW

File: rpcserver.go [hunk 1]
```diff
@@ -5938,19 +5931,14 @@ func (r *rpcServer) GetNodeInfo(ctx context.Context,
...
-			Addresses:  nodeAddrs,
-			Alias:      node.Alias,
-			Color:      routing.EncodeHexColor(node.Color),
-			Features:   features,
-		},
-		NumChannels:   numChannels,
-		TotalCapacity: int64(totalCapacity),
-		Channels:      channels,
-	}, nil
+	return &lnrpc.LightningNode{
```
Comment (guggero) [inline rpcserver.go:5967]: If we're touching this code, can we fix the pre-allocation above (`nodeAddrs`) since we know how many elements there will be?

**Review by guggero — APPROVED** [review]
Nice, LGTM :tada:

File: rpcserver.go (hunk 1 above)
Comment (joostjager) [reply rpcserver.go:5967]: Fixed

**Review by halseth — APPROVED** [review]
Nice, LGTM 👍

**Review by positiveblue — APPROVED** [review]
LGTM :taco:

**Pull request merged**